charts/enterprise-metrics/templates/graphite-querier-dep.yaml

```diff
@@ -0,0 +1,152 @@
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        gossip_ring_member: "true"
+        target: querier
+        release: {{ .Release.Name }}
+        {{- with .Values.graphite_querier.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+{{- if .Values.useExternalConfig }}
+        checksum/config: {{ .Values.externalConfigVersion }}
+{{- else }}
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+{{- end}}
+        {{- with .Values.graphite_querier.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "enterprise-metrics.serviceAccountName" . }}
+    {{- if .Values.graphite_querier.priorityClassName }}
+      priorityClassName: {{ .Values.graphite_querier.priorityClassName }}
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.graphite_querier.securityContext | nindent 8 }}
+      initContainers:
+        {{- toYaml .Values.graphite_querier.initContainers | nindent 8 }}
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - -target=graphite-querier
+            - -config.file=/etc/enterprise-metrics/enterprise-metrics.yaml # TODO(jesus.vazquez) Remove when metrics enterprise v1.6.2 is released
+            - -graphite.enabled=true
+            - -graphite.querier.remote-read-enabled=true
+            - -graphite.querier.query-address=http://{{ template "enterprise-metrics.fullname" . }}-query-frontend.{{ .Release.Namespace }}.svc:{{ .Values.config.server.http_listen_port }}/api/prom
+            - -graphite.querier.graphite-fallback=http://{{ template "enterprise-metrics.fullname" . }}-graphite-web.{{ .Release.Namespace}}.svc:{{ .Values.config.server.http_listen_port }}
+            - -graphite.querier.schemas.default-storage-schemas-file=/etc/graphite-proxy/storage-schemas.conf
+            - -graphite.querier.schemas.default-storage-aggregations-file=/etc/graphite-proxy/storage-aggregations.conf
+            {{- if .Values.minio.enabled }}
+            - -admin.client.backend-type=s3
+            - -admin.client.s3.endpoint={{ .Release.Name }}-minio.{{ .Release.Namespace }}.svc:9000
+            - -admin.client.s3.bucket-name=enterprise-metrics-admin
+            - -admin.client.s3.access-key-id=enterprise-metrics
+            - -admin.client.s3.secret-access-key=supersecret
+            - -admin.client.s3.insecure=true
+            - -blocks-storage.s3.endpoint={{ .Release.Name }}-minio.{{ .Release.Namespace }}.svc:9000
```

Comment:
if the graphite querier uses the remote_read API, does it need to have the blocks storage configured? 
(I'm not sure if the change to make it use the remote_read API has already been merged TBH)

Reply:
>  (I'm not sure if the change to make it use the remote_read API has already been merged TBH)
Yes it has been merged and its about to become the default behaviour.

Removed the blocks storage s3 configuration here https://github.com/grafana/helm-charts/pull/872/commits/174f0bf0b0d25f78dfc2f88e33d209388a93695a